src/apps/applets/domain/applet.py

```diff
@@ -80,6 +80,8 @@ class AppletSingleLanguageDetailMobilePublic(PublicModel):
     )
     encryption: Encryption | None
     stream_enabled: bool | None
+    stream_ip_address: str | None
```

Comment:
maybe it's better to use IPvAnyAddress and PositiveInt ?

Reply:
@ZenaY91 done, pls review again